src/fe-common/core/chat-completion.c

```diff
@@ -1036,9 +1040,11 @@ static char *expand_escapes(const char *line, SERVER_REC *server,
 			/* newline .. we need to send another "send text"
 			   event to handle it (or actually the text before
 			   the newline..) */
-			if (ret != ptr) {
-				*ptr = '\0';
-				signal_emit("send text", 3, ret, server, item);
```

Comment:
Hmm. It's not affecting escapes at all in fe-exec either before or after this change. I guess this is fine.